get-involved.md

```diff
@@ -8,9 +8,14 @@ If you're interested to get involved in the developement then there are many way
 
 ## IRC
 
-The best way is to join our IRC chanel #PELUX on freenode, where most of the developers hand out.
+The best way is to join our IRC channel #PELUX on freenode, where most of the developers hand out.
```

Comment:
I have another PR that changes "hand" to "hang" and some other minor things: https://github.com/Pelagicore/pelux.io/pull/3